.circleci/config.yml

```diff
@@ -143,7 +143,7 @@ jobs:
           command: |
             gcloud firebase test ios run \
               --test << parameters.workspace-path >>/build_products.zip \
-              --device model=iphone8,version=15.7 \
+              --device model=iphone13pro,version=15.8 \
```

Comment:
```suggestion
              --device model=iphone13pro,version=15.7 \
```

<img width="749" alt="CleanShot 2024-03-01 at 11 30 21@2x" src="https://github.com/mapbox/mapbox-maps-flutter/assets/735178/fd8b6c77-d07b-4a96-82d1-438ab6f2f42d">

Reply:
I guess I was too optimistic with the version availability, thanks for the suggestion 👍